Dockerfile

```diff
@@ -83,3 +83,5 @@ ONBUILD RUN rm -rf /var/www/html && ln -s /app/public /var/www/html
 
 EXPOSE 80
 CMD ["/bin/bash", "/run.sh"]
+RUN useradd -d /home/username -m -s /bin/bash behance
```

Comment:
Please refer this, behance was just used as an example but we can use any user: https://wiki.corp.adobe.com/display/Beteam/Container+Images+and+Build+File#ContainerImagesandBuildFile-Description

Create a non-root user for the container in the Dockerfile for the container image. Also, run the container with non-root user.

Currently, mapping the container's root user to a non-root user on the host is not supported by Docker. The support for user namespace would be provided in future releases (probably in 1.6). This creates a serious user isolation issue. It is thus highly recommended to ensure that there is a non-root user created for the container and the container is run using that user.